localstack/services/install.py

```diff
@@ -172,9 +172,11 @@ def install_kinesis_mock():
 
     machine = platform.machine().lower()
     system = platform.system().lower()
+    version = platform.version().lower()
 
     LOG.debug('getting kinesis-mock for %s %s', system, machine)
-    if machine == 'x86_64' or machine == 'amd64':
+    if ((machine == 'x86_64' or machine == 'amd64') and
+        "arm64" not in version and "arm32" not in version):
```

Comment:
could we maybe make the check a bit more explicit, or at least add a comment, that this is a workaround for M1?

something along the lines of
`is_probably_m1 = system == 'darwin' and ('arm64' in version or 'arm32' in version)`

Reply:
Updated